### PR TITLE
[Merged by Bors] - chore(Associated): use `M`, `N` for type variables

### PR DESCRIPTION
**Pull request by urkud**
Replace `α` → `M`, `β` → `N` for type variables.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Associated/Basic.lean [hunk 1]
```diff
@@ -31,20 +31,20 @@ and prove basic properties of this quotient.
 assert_not_exists OrderedCommMonoid
 assert_not_exists Multiset
 
-variable {α : Type*} {β : Type*} {γ : Type*} {δ : Type*}
+variable {M N : Type*}
 
 section Prime
 
-variable [CommMonoidWithZero α]
+variable {M : Type*} [CommMonoidWithZero M]
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Associated/Basic.lean:38]: Why redeclare M here?

**Review by Ruben-VandeVelde — APPROVED** [review]
I have no strong feelings one way or the other on this, but it seems to do what it says on the tin, at least.

File: Mathlib/Algebra/Associated/Basic.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/Algebra/Associated/Basic.lean:38]: AFAIR, I changed this section to `M`, then decided to change it everywhere but forgot to undo this change.
```suggestion
variable [CommMonoidWithZero M]
```

**Comment by j-loreaux** [comment]
sure, why not

bors merge

**Pull request closed without merging**